pandas/tests/sparse/test_series.py

```diff
@@ -87,6 +87,12 @@ def setup_method(self, method):
                                       fill_value=0)
         self.ziseries2 = SparseSeries(arr, index=index, kind='integer',
                                       fill_value=0)
+    def test_constructor_data_input(self):
+        arr = SparseSeries({1: 1})
+        assert arr.count() == len({1: 1})
+
+        arr = SparseSeries(pd.Series({1: 1}, index=[0, 1, 2]))
+        assert arr.count() == pd.Series({1: 1}, index=[0, 1, 2]).count()
```

Comment:
I would separate out your `pd.Series({1: 1}, index=[0, 1, 2]).count()` as a variable like "expected" so that it's a little clearer what the expected output is.